Add --toggle command-line flag and current time display

ovehbe opened the pull request:
This commit adds two new features:

1. Command-line toggle functionality:
   - Added --toggle flag to show/hide the window via command line
   - Useful for keyboard shortcuts and external scripts
   - Usage: flatpak run io.github.eminfedar.vaktisalah-gtk-rs --toggle
   - Modified src/main.rs to handle command-line arguments

2. Current time display:
   - Added live current time (HH:MM format) below the dates
   - Styled consistently with the rest of the UI
   - Updates every second
   - Automatically adapts to light/dark mode
   - Modified src/window/imp.rs, src/window/mod.rs, and ui/MainWindow.blp

<img width="402" height="582" alt="Screenshot from 2025-11-09 14 35 29" src="https://github.com/user-attachments/assets/04a9d5ad-47b7-4d80-b745-5f73ddc527d0" />

ovehbe committed:
Add --toggle command-line flag and current time display

This commit adds two new features:

1. Command-line toggle functionality:
   - Added --toggle flag to show/hide the window via command line
   - Useful for keyboard shortcuts and external scripts
   - Usage: flatpak run io.github.eminfedar.vaktisalah-gtk-rs --toggle
   - Modified src/main.rs to handle command-line arguments

2. Current time display:
   - Added live current time (HH:MM format) below the dates
   - Styled consistently with the rest of the UI
   - Updates every second
   - Automatically adapts to light/dark mode
   - Modified src/window/imp.rs, src/window/mod.rs, and ui/MainWindow.blp

eminfedar commented:
Thanks for your nice contribution.

It is a very useful addition. I will return to you after I checked and tested the PR. It seems nice in first sight.

eminfedar commented on ui/MainWindow.blp:
Can you change `default-height` property from 460 to 510 to fit all prayer times on the window at startup?

After this change, I will merge this. Thanks for your efforts again.

ovehbe replied:
Sorry for the late response, I have changed the property as per requested.

ovehbe committed:
Increase default window height to 510px to fit all prayer times

Changed default-height from 460 to 510 to ensure all prayer times are
visible at startup without scrolling, as requested by maintainer.
